src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] The main component `App.js` is stateless. Kindly change the component to a **class/ stateful** component